tests/test_fail_messages.py

```diff
@@ -93,24 +93,6 @@
      "moo' was found in 'echo does not contain moo': stdout while "
      "it should not be there"),
     ("""\
-    - name: fail_test
-      command: grep
-      stderr:
-        contains:
-          - "No arguments?"
-    """,
```

Comment:
This test should not be removed. It appropriately tests the "contains" error message. Please find away to get this test to run succesfully.